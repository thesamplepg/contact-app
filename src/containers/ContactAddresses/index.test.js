import React from "react";
import { shallow } from "enzyme";

import { TestComponent } from "./index";

describe("ContactAddresses Component", () => {
  const props = { contacts: [] };

  const renderComponent = props => shallow(<TestComponent {...props} />);

  it("render propertly", () => {
    expect(renderComponent(props)).toMatchSnapshot();
  });
});
