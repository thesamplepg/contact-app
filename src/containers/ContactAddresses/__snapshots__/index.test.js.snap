// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactAddresses Component render propertly 1`] = `
<section
  className="ContactAddresses"
>
  <ContactsList
    contacts={Array []}
  />
</section>
`;
